src/test/regress/expected/multi_replicate_reference_table.out

```diff
@@ -360,6 +360,7 @@ SELECT create_distributed_table('replicate_reference_table_hash', 'column1');
  
 (1 row)
 
+UPDATE pg_dist_partition SET repmodel='c' WHERE logicalrelid='replicate_reference_table_hash'::regclass;
```

Comment:
It would be greate if you add a single line comment why we're updating the repmodel here